examples/04_get_balance.rs

```diff
@@ -4,13 +4,34 @@
 //! cargo run --example get_balance --release
 // In this example we sync the account and get the balance
 // Rename `.env.example` to `.env` first
+use std::{env, path::PathBuf};
 
-use iota_wallet::{account_manager::AccountManager, Result};
+use dotenv::dotenv;
+use iota_wallet::{
+    account_manager::AccountManager,
+    iota_client::constants::SHIMMER_COIN_TYPE,
+    secret::{stronghold::StrongholdSecretManager, SecretManager},
+    ClientOptions, Result,
+};
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
+    dotenv().ok();
+
+    let client_options = ClientOptions::new().with_node(&env::var("NODE_URL").unwrap())?;
+
+    let secret_manager = StrongholdSecretManager::builder()
+        .password(&env::var("STRONGHOLD_PASSWORD").unwrap())
+        .build(PathBuf::from("wallet.stronghold"))?;
+
     // Create the account manager
-    let manager = AccountManager::builder().finish().await?;
+    let manager = AccountManager::builder()
+        .with_client_options(client_options)
+        .with_coin_type(SHIMMER_COIN_TYPE)
+        .with_secret_manager(SecretManager::Stronghold(secret_manager))
+        .finish()
+        .await?;
```

Comment:
Why all these changes ?

Reply:
The example doesnt run without them. not sure why we didnt add

Reply:
It runs fine, but the examples are sequential, you need to have run at least 01 before. None of the examples have that so let's revert.